code/game/turfs/simulated/floor.dm

```diff
@@ -30,10 +30,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	var/broken = 0
 	var/burnt = 0
 	var/floor_tile = null //tile that this floor drops
-	var/list/broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
-	var/list/burnt_states = list()
+	var/list/broken_states
+	var/list/burnt_states
 
 /turf/open/floor/New()
```

Comment:
Is there any reason we can't move this elsewhere, or have the mapper take care of this by adding a few more subtypes of turfs, because removing this will remove a few more proc calls.